keras_nlp/models/bert/bert_presets.py

```diff
@@ -34,6 +34,14 @@
             "Tiny size of BERT where all input is lowercased. "
             "Trained on English Wikipedia + BooksCorpus."
         ),
+        "metadata":{
+            "description": (
+                "Tiny size of BERT where all input is lowercased. "
+                "Trained on English Wikipedia + BooksCorpus."
+            ),
+            "params": "4M",
```

Comment:
We should keep a uniform style here... The nicest thing might be to keep the param count as you do above, and on keras.io write a utility to truncate, e.g. 222595584 -> "222M"